example-deck.gl.md

```diff
@@ -0,0 +1,400 @@
+Plotting CO-Values using deck.gl
+================================
+
+In this example we explain how to use [deck.gl](https://deck.gl/#/),
+a WebGL-powered framework for data visualization created by [Uber](https://www.uber.com),
+to visualize the amount of carbon monoxide in Germany from a couple of days.
+
+The result should look somewhat like this:
+
+<style type="text/css">
+      #container {width: 100%; height: 500px;}
+    </style>
+<div id="container"></div>
+
+We are mostly following the [scripting get-started](https://github.com/uber/deck.gl/blob/7.3-release/examples/get-started/scripting/mapbox/index.html) from deck.gl.
+
+## GeoJSON Data
+
+Let's take a look at the data first.
+We request all measured carbon monoxide values from Germany from the fourth and fifth of February (this time span is arbitrary chosen) using the [geo.json](https://demo.emissions-api.org/ui/#/default/emissionsapi.web.get_data) API endpoint and the url query parameters *begin*, *end* and *polygon*:
+
+```
+https://demo.emissions-api.org/api/v1/geo.json
+    ?begin=2019-02-04
+    &end=2019-02-06
+    &polygon=9.921906365609232,54.983104153048025,9.9395797054529,54.596641954153256,…,9.921906365609232,54.983104153048025
```

Comment:
If it's the bounding box of Germany anyway:
```suggestion
    &country=DE
```

Reply:
It is not a bounding box. It is a polygon describing the vague form of Germany. Maybe we can change this later, if `DE` not describing a rectangle anymore.